### DISCUSSION TITLE
Access free GitHub Education benefits

### DISCUSSION THREAD

**Discussion by thiagocorreanet** [community/community#142295] [GitHub Education]
### Hi everyone,

Hello, I was approved more than 5 days ago, and the plan has not been activated yet.

![image](https://github.com/user-attachments/assets/67a66def-9c17-4526-a7a5-5f2c724d58de)

**Comment by davevad93** [comment]
Hi @thiagocorreanet ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @thiagocorreanet was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
